rust/worker/src/execution/operators/knn.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug)]
+pub struct KnnOperator {
+    pub embedding: Vec<f32>,
+    pub fetch: u32,
```

Comment:
should this be `limit`?

Reply:
I'm using `fetch` in consistence with the limit operator here:

https://github.com/chroma-core/chroma/blob/834c9dcf6a56c6209a40f5985c59431a21d74138/chromadb/execution/expression/operator.py#L27-L30

which is borrowed from datafusion [here](https://docs.rs/datafusion/latest/datafusion/logical_expr/logical_plan/struct.Limit.html)